Firestore/core/src/api/firestore.cc

```diff
@@ -191,6 +191,10 @@ void Firestore::DisableNetwork(util::StatusCallback callback) {
   client_->DisableNetwork(std::move(callback));
 }
 
+void Firestore::SetClientLanguageHeader(std::string language_header) {
+  FirestoreClient::SetClientLanguageHeader(std::move(language_header));
```

Comment:
I see little reason for all the additional plumbing of the value of this property. It seems reasonable to have `Firestore` directly call the setter on `GrpcConnection`. I do see value in having this be a part of `Firestore`: this makes the core API better defined and gives us flexibility in how/where this is implemented in here without having to update the public C++ code.

Reply:
Previously `Firestore` was unaware of the "remote" layer, but I agree this is more of an academic concern. Changed.